1_fetch.R

```diff
@@ -357,7 +378,22 @@ p1_targets_list <- list(
                list.files(path = dirname(p1_natural_baseflow_zip),
                           pattern = "*baseflow.*.csv"))
       },
-    format = "file"
+    format = "file",
+    repository = 'local'
+  ),
+
+  # CSV file of gridmet drivers aggregated to PRMS segments
+  tar_target(
+    p1_gridmet_csv,
+    "1_fetch/in/drb_climate_2022_04_06_segments.csv",
```

Comment:
How large is this gridmet file? Would it be better to host it on S3?

Reply:
Sorry, I meant to store the gridmet csv file on S3

Reply:
I guess I'm not sure what you mean. Because it's a target, it _is_ stored on S3:

![image](https://user-images.githubusercontent.com/6943441/170534664-601ba500-8eca-45d5-a298-ff89f671c109.png)

Is that what you mean? Or were you thinking of something else?